pkg/services/auth/idimpl/service.go

```diff
@@ -98,12 +115,37 @@ func (s *Service) SignIdentity(ctx context.Context, id identity.Requester) (stri
 	return result.(string), nil
 }
 
+func (s *Service) setUserClaims(ctx context.Context, identifier string, claims *auth.IDClaims) error {
+	id, err := strconv.ParseInt(identifier, 10, 64)
```

Comment:
this identifier could be an API Key ID

Reply:
It is done on call site https://github.com/grafana/grafana/pull/80622/files#diff-6e6c983607a83236863ab4de3310f47d4f0f7cb1a3ce9a2b15b42070da78a2e7R92-R96

Reply:
:+1: sorry, didn't see it